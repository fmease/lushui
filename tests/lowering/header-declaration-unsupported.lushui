;; TEST fail check

; module-level
; documentation
; here
module =

@foreign
module =
