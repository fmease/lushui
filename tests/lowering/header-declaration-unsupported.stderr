error: module headers not supported yet
 --> ${DIRECTORY}/lowering/header-declaration-unsupported.lushui:6:1
  |
6 | module =
  | ^^^^^^ 
  |

error[E013]: attribute `foreign` cannot be ascribed to a module header declaraiton
 --> ${DIRECTORY}/lowering/header-declaration-unsupported.lushui:8:1
  |
8 | @foreign
  | ^^^^^^^^ misplaced attribute
  |
 --> ${DIRECTORY}/lowering/header-declaration-unsupported.lushui:9:1
  |
9 | module =
  | ------ incompatible item
  |
 note: attribute `foreign` can only be ascribed to value or data declarations

error: module headers not supported yet
 --> ${DIRECTORY}/lowering/header-declaration-unsupported.lushui:9:1
  |
9 | module =
  | ^^^^^^ 
  |

error: aborting due to 3 previous errors

